gifski-api/src/encodegifsicle.rs

```diff
@@ -76,7 +76,7 @@ impl Encoder for Gifsicle<'_> {
             };
             gfs.screen_width = image.width() as _;
             gfs.screen_height = image.height() as _;
-            gfs.loopcount = if settings.once {1} else {0}; // 0 means infinite
+            gfs.loopcount = if settings.once { 1 } else { -1 };
```

Comment:
@kornelski The change from `0` to `-1` was intentional, right?

Reply:
Good catch. That's a regression. I didn't backport the fix from the app. Fixed in master now.